apps/package.json

```diff
@@ -139,7 +139,7 @@
     "node-sass": "^4.0.0",
     "npm-which": "2.0.0",
     "phantomjs-prebuilt": "^2.1.3",
-    "playground-io": "0.4.0",
+    "playground-io": "code-dot-org/playground-io",
```

Comment:
I haven't bothered npm publishing this fork, so I suppose I'll just add Code.org-specific tags to my version commits (I always bump the package.json version in its own commit).